flutter_src/lib/logic/state/bloc_validator.dart

```diff
@@ -31,12 +48,14 @@ class CustomBlocValidator {
     List<String> emails = string.split(',');
 
     for (int i = 0; i < emails.length; i++) {
-      bool emailValid = RegExp(r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+").hasMatch(emails[i]);
+      bool emailValid = RegExp(
+              r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+")
+          .hasMatch(emails[i]);
       if (!emailValid) {
         errors.add(emails[i] + " is not an valid email.");
```

Comment:
no need to be so specific, still better to return just false and have a generic failure message.